MekHQ/src/mekhq/campaign/parts/Refit.java

```diff
@@ -2101,12 +2101,12 @@ private static void processArmorSupplies(Refit retVal, Node wn, Version version)
     }
 
     public void reCalc() {
-        setCampaign(getCampaign());
+        setCampaign(oldUnit.getCampaign());
```

Comment:
With the addition of setCampaign in fixReferences this is not necessary at this time, although I don't know if it may come up in future.